src/ostorlab/agent/message/proto/v3/fingerprint/file/ios/backend/backend.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.fingerprint.file.ios.backend;
+
+message Location {
+  optional int32 version = 1 [default = 4];
```

Comment:
The version attribute should be inside the IP message, not the location message.